lms/static/sass/base/_layouts.scss

```diff
@@ -0,0 +1,104 @@
+// base layout styles to support early responsive lms
+// may be discarded later once sass breakpoints are wired in
+
+body.view-incourse {
```

Comment:
NIT: can you add a comment giving context that these rules all pertain to a student being "in a course"?